CHANGELOG.md

```diff
@@ -144,6 +144,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 	`DrawLimbPathVisualizations` - any  `AHumans` or `ACrabs` will draw some of their `LimbPaths` to the standard view.  
 	`DrawRayCastVisualizations` - any rays cast by `SceneMan` will be drawn to the standard view.  
 	`DrawPixelCheckVisualizations ` - any pixel checks made by `SceneMan:GetTerrMatter` or `SceneMan:GetMOIDPixel` will be drawn to the standard view.
+	
+- A fully featured inventory view for managing `AHumans'` inventories (to be expanded to other things in future).
```

Comment:
`AHumans'` inventories means multiple inventories belonging to multiple AHumans cause the apostrophe is after the `s` in AHumans rather than before it. But sure, I can change it to remove the possessive if it's confusing.